Lesson6 Add OAuth(GitHub, Vk, Google)

Skurik-SA opened the pull request.

Skurik-SA committed:
Lesson6: Add OAuth

Skurik-SA committed:
Lesson6: Add OAuth (github, vk, google)

chrn-feip commented on fefu-app2/app/Models/User.php:
```diff
@@ -61,6 +94,11 @@ class User extends Authenticatable
+        'github_id',
+        'vkontakte_id',
+        'google_id',
+        'app_logged_in_at',
```
Зачем делать скрытым это и следующее поля?

chrn-feip commented on fefu-app2/app/Http/Resources/UserResource.php:
```diff
@@ -0,0 +1,28 @@
+     */
+    public function toArray($request)
+    {
+        return [
```
Не хватает полей  app_*

chrn-feip commented on fefu-app2/resources/views/auth/profile.blade.php:
```diff
@@ -7,6 +7,28 @@
 <body>
 <h1>Profile</h1>
+<div>
+    <h3>OAuth info:</h3>
```
Надо ещё вывести информацию про app_* поля

Skurik-SA committed:
fixed

Я убрал из hidden поля. Их не нужно скрывать потому что они всем известны и никому никакой ненужной пользы не принесут?
И добавил время входа и регистрации для обычной авторизации.
